app/build.gradle

```diff
@@ -80,6 +80,10 @@ dependencies {
         transitive = true;
     }
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.6.2'
```

Comment:
consider using leak canary in release in order to include it in QA tests

Reply:
QA tests on debug first :)